core/src/main/java/com/linecorp/armeria/common/logging/RequestLog.java

```diff
@@ -266,19 +267,29 @@ default void ensureAvailability(Iterable<RequestLogAvailability> properties) {
     String query();
 
     /**
-     * Returns the status code specific to the {@link Response} of the current {@link SessionProtocol}.
+     * Returns the status of the {@link Response}.
      *
      * @throws RequestLogAvailabilityException if this property is not available yet
      */
-    int statusCode();
+    default HttpStatus status() {
```

Comment:
Reviewed the nullability of all methods and parameters and added `@Nullable` to the eligible params and methods.